config/charts/inferencepool/templates/epp-deployment.yaml

```diff
@@ -27,6 +27,45 @@ spec:
       # Conservatively, this timeout should mirror the longest grace period of the pods within the pool
       terminationGracePeriodSeconds: 130
       containers:
+      {{- if .Values.inferenceExtension.sidecar.enabled }}
+      - name: sidecar
```

Comment:
instead of hardcoding the name, make it `.Values.inferenceExtension.sidecar.name`